### DISCUSSION TITLE
[Feature] support vectorcord,  the successor of pgvecto.rs.

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#14997] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

https://github.com/tensorchord/VectorChord

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Thank you. We’re aware of this and will be upgrading when appropriate.
